variables.tf

```
@@ -207,6 +207,21 @@ variable "share_retention_policy_days" {
   }
 }
 
+variable "share_cors_rules" {
+  description = "A list of CORS rules to configure for this File Share."
```

Update phrasing to be similar to `blob_cors_rules` description:

```suggestion
  description = "A list of CORS rules to configure for this File Storage."
```